null_safety_examples/misc/lib/language_tour/typedefs/misc.dart

```diff
@@ -1,6 +1,13 @@
-// #docregion Function
-typedef F = List<T> Function<T>(T);
-// #enddocregion Function
+// #docregion int-list
+typedef IntList = List<int>;
+IntList il = [1,2,3];
+// #enddocregion int-list
+
+// #docregion list-mapper
+typedef ListMapper<X> = Map<X, List<X>>;
+Map<String, List<String>> m1; // Verbose.
+ListMapper<String> m2; // Same thing but less verbose, less redundant.
```

Comment:
I might replace ", less redundant" with "and with clearer intent." or something to that effect.  That is, the conciseness is a bonus, but I also think that the named version expresses intent in the code in a way that makes it much more readable.